company-profiles/atomic-squirrel.md

```diff
@@ -0,0 +1,29 @@
+# Atomic Squirrel
+
+## Company blurb
+
+Atomic Squirrel specializes in helping startups get off the ground.
+
+## Company size
+
+Team of 2.
```

Comment:
@brikis98 - when this is merged, can you edit this profile to fill out the information please? Thanks